examples/wagmi-v2/package.json

```diff
@@ -19,7 +19,7 @@
 		"@oasisprotocol/sapphire-wagmi-v2": "workspace:^",
 		"@playwright/test": "^1.48.2",
 		"@tanstack/react-query": "5.0.5",
-		"@tenkeylabs/dappwright": "^2.10.0",
+		"@tenkeylabs/dappwright": "^2.11.0",
```

Comment:
AFAIK dappwright and playwright should probably go under `devDependencies`? Since you don't need those during execution or when someone else is importing this package (unlikely though).